install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -323,6 +323,17 @@ rules:
       - list
       - patch
 
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - create
+      - watch
+      - get
+      - list
+      - patch
+
```

Comment:
Are we creating Services across namespaces? Should this be in the `Role` rather than the `ClusterRole`? Curious that we don't already have this, I thought we were creating a service from MAO already 🤔

Reply:
I didn't find any cases where the MAO was creating services from go code. There's one in a manifest file.

I've moved this block up to the Role instead of the ClusterRole because the new service is just created in the openshift-machine-api namespace.